new_platforms/docs/ta_debugging_wsl.md

```diff
@@ -0,0 +1,139 @@
+Debugging OP-TEE TAs on WSL
+===========================
+
+This document is a complement to [Debugging OP-TEE TAs with
+QEMU](ta_debugging_qemu.md). That guide assumes it is being followed on a native
+Linux system. This guide shows you how to configure the Windows Subsystem for
+Linux (WSL) to use the guide on WSL.
+
+In this guide, you will learn how to:
+
+1. Install and configure a native X Server;
+2. Work around a UNIX feature used by `fakeroot` not yet implemented in WSL.
```

Comment:
Is it really not yet or not? Just curious if we know WSL is planning on supporting it.

Reply:
https://github.com/Microsoft/WSL/issues/1443